eng/clipack/Common.projitems

```diff
@@ -48,8 +49,12 @@
         Properties="@(AdditionalProperties)"
         RemoveProperties="OutputPath;TargetFramework" />
 
+    <PropertyGroup>
+      <_OutputBinaryPath>$(OutputPath)/aspire</_OutputBinaryPath>
+    </PropertyGroup>
+
     <!-- TODO: avoid generating the file instead of manually deleting it here -->
-    <Delete Files="$(OutputPath)\aspire.xml" Condition="Exists('$(OutputPath)\aspire.xml')" />
+    <Delete Files="$(_OutputBinaryPath).xml" Condition="Exists('$(_OutputBinaryPath).xml')" />
```

Comment:
```suggestion
    <Delete Files="$(OutputPath)\aspire.xml" Condition="Exists('$(OutputPath)\aspire.xml')" />
```

This could be reverted now. The name isn't that great because on Windows the binary path is "aspire.exe", so it doesn't really fit.